recipes-go/3mdeb-rtectrl/3mdeb-rtectrl_git.bbappend

```diff
@@ -1,13 +1,23 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
+check_distro_ver() {
```

Comment:
This is not needed. The script is in the same repo as distro. Once it's not needed anymore, we will simply drop it.

Reply:
Dropped the commit [here](https://github.com/3mdeb/meta-rte/compare/c4c0f2b6c947061defa1b83bd585745ec0ecadb6..b09e9b726f3372dd7e3b9054bc79468968f7ece0)